#2569 Fixed deleting non existent user profile

Patrykb0802 opened the pull request:
- Added condition for showing delete button in showUserProfile function in UserProfiles.jsp
- Deprecated show($("deleteButton")); that forced showing delete button no matter what.
- Added show("deleteButton"); to saveUserProfile function to show deletebutton for freshly made user profiles

Commit:
#2569 Fixed deleting non existent user profile
- Added condition for showing delete button in showUserProfile function in UserProfiles.jsp
- Deprecated show($("deleteButton")); that forced showing delete button no matter what.
- Added show("deleteButton"); to saveUserProfile function to show deletebutton for freshly made user profiles

Limraj commented on WebContent/WEB-INF/jsp/usersProfiles.jsp:
```diff
@@ -133,6 +133,10 @@
     }
     
     function showUserProfile(userProfileId) {
+        if (userProfileId == -1)
```
1. Click on 'Add user profile';
2. Empty 'Profile name';
3. Click on 'save'; -> Message appears 'Profile name must be unique and not empty'
4. "Delete" button appears;
5. Click on "Delete" button -> "Are you sure you want to delete this profil?" ok -> "Profile successfully deleted." ok -> "Failed to execute 'removeChild' on 'Node': parameter 1 is not of type 'Node'." error

Commit:
#2569 Fixed deleting non existent user profile
- Added condition for showing delete button in showUserProfile function in UserProfiles.jsp
- Deprecated show($("deleteButton")); that forced showing delete button no matter what.
- Added show("deleteButton"); to saveUserProfileCB function to show deletebutton for freshly made user profiles
